### DISCUSSION TITLE
Link (next/link) component

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#37246] [Help]
i cant get Link to work for the life of me 

Heres my code:
![image](https://user-images.githubusercontent.com/98897812/170731440-fbafc49f-0d29-4cb4-b8a2-7b35c3b6dbc5.png)

my end goal is to use it with navbar , i cant even get it to work so it might not work for me

(yes i know there arnt quote marks where there should be in the image i fixed that)

**Comment by ghost** [comment]
[vercel-logs](https://pastebin.com/GpNGPkRp)

**Comment by icyJoseph** [comment]
The prop `href` from Link is misspelled as `herf`.

Is that the problem perhaps? 

The paste bin seems to show other issue though, hard to tell on mobile view.
